.isort.cfg

```diff
@@ -10,3 +10,4 @@ known_odoo=odoo
 known_odoo_addons=odoo.addons
 sections=FUTURE,STDLIB,THIRDPARTY,ODOO,ODOO_ADDONS,FIRSTPARTY,LOCALFOLDER
 default_section=THIRDPARTY
+known_third_party = dateutil,lxml,mock,psycopg2,setuptools
```

Comment:
I'm not sure this is required after `default_section` has been changed.

Reply:
it was generated by pre-commit, otherwise fails